apps/fly-my-rockets/src/app/rockets/rocket-show/rocket-show.component.html

```diff
@@ -5,13 +5,19 @@
     </mat-card-header>
     <img mat-card-image *ngIf="rocketPhotoUrl$ | async as photoUrl" [src]="photoUrl" [alt]="rocket.name">
     <mat-card-actions>
-      <button mat-button [matMenuTriggerFor]="rocketEdit">Edit</button>
-      <button mat-button color="warn" (click)="deleteRocket()">Delete</button>
+      <button mat-button (click)="openRocketDialog(rocket)">
```

Comment:
Went back and forth on having the edit and remove photo buttons be directly on the card vs in the menu, but having them directly on the card is more consistent with the Material Design specs. See https://www.material.io/components/cards#actions.